biogeochem/FatesCohortMod.F90

```diff
@@ -142,13 +142,15 @@ module FatesCohortMod
     real(r8) :: gpp_acc
     real(r8) :: gpp_acc_hold
 
-    real(r8) :: npp_tstep                 ! Net Primary Production (see above *)
     real(r8) :: npp_acc
     real(r8) :: npp_acc_hold
 
-    real(r8) :: resp_tstep                ! Autotrophic respiration (see above *)
-    real(r8) :: resp_acc
-    real(r8) :: resp_acc_hold
+    real(r8) :: resp_m_tstep              ! Maintenance respiration (see above *)
+    real(r8) :: resp_m_acc
+    real(r8) :: resp_m_acc_hold
+
+    real(r8) :: resp_g_acc                ! Growth respication can only be calculated at the daily timestep
```

Comment:
@ckoven , do we need resp_g_acc?  The <var>_acc variable was created to track the accumulation of the flux through the fast timesteps over the course of the day. Since we don't accumulate growth respiration anymore, since it is defined only at the end of the day, I believe we only need resp_g_acc_hold, right?

Reply:
right, this was a thing I wasn't sure about. I was guessing that it is as you describe, but wanted to include the `_acc` variable to be on the safe side. If you don't think it is needed, then happy to remove.